stripe/src/main/resources/templates/checkout.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html xmlns='http://www.w3.org/1999/xhtml' xmlns:th='http://www.thymeleaf.org'>
+    <head>
+        <title>Checkout</title>
+        <style>
+            body {
+                font-family: 'arial';
+            }
+            #checkout-form input,
+            #checkout-form button {
+                display: block;
+                margin: 12px;
+            }
+        </style>
+    </head>
+    <body>
+        <form action='/charge' method='POST' id='checkout-form'>
+            <input type='hidden' th:value='${amount}' name='amount' />
+            <label>Price:<span th:text='${amount/100}' /></label>
+            <!-- NOTE: data-key/data-amount/data-currency will be rendered by Thymeleaf -->
+            <script
```

Comment:
Although it is supported elsewhere now, It's more typical to have a <script> tag in the <head> section. 
Either way, you do need a closing </script> tag instead of the shortcut <script ...  />